assets/panel/debugger.properties

```diff
@@ -237,6 +237,14 @@ breakpointMenuItem.disableAll.accesskey=k
 breakpointMenuItem.deleteAll=Remove all breakpoints
 breakpointMenuItem.deleteAll.accesskey=a
 
+# LOCALIZATION NOTE (panel.sources): Label displayed in the primary
+# pane. %S is replaced by the string sources.
+panel.sources=Sources
```

Comment:
looks like already have this string: https://github.com/pradeepgangwar/debugger.html/blob/ed91f18bda935b6bc157413bbe71c6f427b3e1ab/assets/panel/debugger.properties#L434

Reply:
As I wrote a few days ago in Bugzilla, adding localization comments is not mandatory, they need to be helpful. In this case, try reading the localization comment and tell me if it matches the string (same for the next one) ;-)

@jasonLaster 
String reuse in different contexts should be avoided. Let's keep this separate from the one use from the header.